src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.width = height;

```

Comment:
There is a mistake in the constructor: `this.width = height;` should be `this.height = height;`. This error causes the `height` field to be incorrectly initialized, affecting the area calculation.